### ISSUE TITLE
Add Prometheus metrics

### ISSUE THREAD

**Issue by a60814billy** [hackmdio/codimd#1466]
The monitoring metrics is important in cloud native era. we should provide prometheus metrics instead of status page.

**Comment by tarlety** [comment]
How about exporting metrics like this?

### How Metrics Looks Like

1. **/metrics/router** : exporting node.js/express Prometheus metrics by [prometheus-api-metrics](https://www.npmjs.com/package/prometheus-api-metrics)

![Screenshot from 2020-04-29 17-37-59](https://user-images.githubusercontent.com/18380640/80581752-34681d80-8a40-11ea-952d-9d06432b3f52.png)

2. **/metrics/codimd** : exporting codimd realtime status (/status) as Prometheus metrics

![Screenshot from 2020-04-29 17-34-10](https://user-images.githubusercontent.com/18380640/80581374-a68c3280-8a3f-11ea-963d-6233d1c5e59f.png)

### How Prometheus Configuration Looks Like

```
- job_name: codimd-router
  honor_timestamps: true
  scrape_interval: 5s
  scrape_timeout: 5s
  metrics_path: /metrics/router
  scheme: http
  static_configs:
  - targets:
    - localhost:3001
- job_name: codimd
  honor_timestamps: true
  scrape_interval: 5s
  scrape_timeout: 5s
  metrics_path: /metrics/codimd
  scheme: http
  static_configs:
  - targets:
    - localhost:3001
```

### How Grafana Dashboard Looks Like

Online Notes and Online Users and other stuff:

![Screenshot from 2020-04-29 17-58-12 resized](https://user-images.githubusercontent.com/18380640/80583633-0e904800-8a43-11ea-8b7d-c7342407298f.png)

95th Response Time by specific status code:

![Screenshot from 2020-04-29 18-11-19](https://user-images.githubusercontent.com/18380640/80584811-d984f500-8a44-11ea-88e2-497357202a08.png)

**Comment by tarlety** [comment]
If this looks fine, I will fire a PR these days.

**Comment by BrooklynQ1978** [comment]
> How about exporting metrics like this?
> 
> ### How Metrics Looks Like
> 1. **/metrics/router** : exporting node.js/express Prometheus metrics by [prometheus-api-metrics](https://www.npmjs.com/package/prometheus-api-metrics)
> 
> ![Screenshot from 2020-04-29 17-37-59](https://user-images.githubusercontent.com/18380640/80581752-34681d80-8a40-11ea-952d-9d06432b3f52.png)
> 
> 1. **/metrics/codimd** : exporting codimd realtime status (/status) as Prometheus metrics
> 
> ![Screenshot from 2020-04-29 17-34-10](https://user-images.githubusercontent.com/18380640/80581374-a68c3280-8a3f-11ea-963d-6233d1c5e59f.png)
> 
> ### How Prometheus Configuration Looks Like
> ```
> - job_name: codimd-router
>   honor_timestamps: true
>   scrape_interval: 5s
>   scrape_timeout: 5s
>   metrics_path: /metrics/router
>   scheme: http
>   static_configs:
>   - targets:
>     - localhost:3001
> - job_name: codimd
>   honor_timestamps: true
>   scrape_interval: 5s
>   scrape_timeout: 5s
>   metrics_path: /metrics/codimd
>   scheme: http
>   static_configs:
>   - targets:
>     - localhost:3001
> ```
> 
> ### How Grafana Dashboard Looks Like
> Online Notes and Online Users and other stuff:
> 
> ![Screenshot from 2020-04-29 17-58-12 resized](https://user-images.githubusercontent.com/18380640/80583633-0e904800-8a43-11ea-8b7d-c7342407298f.png)
> 
> 95th Response Time by specific status code:
> 
> ![Screenshot from 2020-04-29 18-11-19](https://user-images.githubusercontent.com/18380640/80584811-d984f500-8a44-11ea-88e2-497357202a08.png)

**Comment by a60814billy** [comment]
resolved in #1507
